Sources/ScipioKit/BuildOptions.swift

```diff
@@ -1,7 +1,20 @@
 import Foundation
 import OrderedCollections
+import Basics
 
 struct BuildOptions: Hashable, Codable, Sendable {
+
+    private enum CodingKeys: CodingKey {
+        case buildConfiguration
+        case isDebugSymbolsEmbedded
+        case frameworkType
+        case sdks
+        case extraFlags
+        case extraBuildParameters
+        case enableLibraryEvolution
+        case customFrameworkModuleMapContents
+    }
```

Comment:
Can you tell me why did you add the coding keys?

We already use Codable without CodingKeys using `KeyEncodingStrategy`, so we're better to avoid adding `CodingKey` it was not required.

Reply:
I added `CodingKeys` to exclude the `environment` property from the encoding process. If you remove it the `testEncodeCacheKey()` test would fail as it would contain the environment in the output JSON:
```
"buildOptions" : {
    "buildConfiguration" : "release",
    "enableLibraryEvolution" : true,
    "environment" : {
      "DEVELOPER_DIR" : "\/Xcode.app\/Contents\/Developer",
      "PATH" : "\/Xcode.app\/Contents\/Developer:\/usr\/bin:\/bin"
    },
    ...
```